web-admin/src/components/authentication/UserButton.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+  import SimpleActionMenu from "@rilldata/web-common/components/menu/wrappers/SimpleActionMenu.svelte";
+  import { useAdminServiceGetCurrentUser } from "../../client";
+  import { ADMIN_URL } from "../../client/http-client";
+
+  const user = useAdminServiceGetCurrentUser();
+
+  function handleLogOut() {
+    window.location.href = `${ADMIN_URL}/auth/logout?redirect=${window.origin}`;
+  }
+
+  const isDev = process.env.NODE_ENV === "development";
+</script>
+
+<SimpleActionMenu
+  options={[{ main: "Logout", callback: handleLogOut }]}
+  let:toggleMenu
+  minWidth="0px"
+  distance={4}
+>
+  <img
+    src={$user.data?.user?.photoUrl}
+    alt="avatar"
+    class="h-7 rounded-full cursor-pointer"
+    referrerpolicy={isDev ? "no-referrer" : ""}
```

Comment:
Why `no-referrer` here for dev? Wondering if it's needed in prod

Reply:
The commentary [here on Stack Overflow](https://stackoverflow.com/questions/40570117/http403-forbidden-error-when-trying-to-load-img-src-with-google-profile-pic) implied it was just an issue with "localhost"

Reply:
Not yet sure if it'll be an issue in prod. Even if it's not an issue in prod, we could remove the conditional to keep it simple?

Reply:
Interesting – let's keep the conditional and see if it's a problem in prod